src/app/_types/markdown.d.ts

```diff
@@ -1,11 +1,11 @@
 declare module '../content/*.md' {
   const attributes: import('@/types/attributesTypes').SecurityMaturityModelAttributes
   const react: React.ComponentType
-  export { attributes, react }
+  export { attributes, body, react }
 }
 
 declare module '*.md' {
   const attributes: import('@/types/attributesTypes').PageAttributes
   const react: React.ComponentType
-  export { attributes, react }
+  export { attributes, body, react }
```

Comment:
We might not need this anymore, depending on what you think about [this PR](some of the work on https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/801)

Nothing to change here, just flagging.